tools/clang/test/SemaHLSL/template-checks.hlsl

```diff
@@ -13,19 +13,9 @@ matrix<bool, 1, 2> m_bool;
 // vector<float, 0> v_float_small;
 // vector<SamplerState, 2> v_obj_sampler;
 
-struct s_float2_float3
-{
-  float2 f_float2;
-  float3 f_float3;
-};
-RWBuffer<s_float2_float3> rwb_struct;           /* fxc-error {{X3037: elements of typed buffers and textures must fit in four 32-bit quantities}} */
+RWBuffer<double3> rwb_struct;    /* expected-error {{Elements of typed buffers and textures must fit in four 32-bit quantities}} fxc-error {{X3037: elements of typed buffers and textures must fit in four 32-bit quantities}} */
 
-struct s_float4_sampler
-{
-  float4 f_float4;
-  SamplerState f_sampler;
-};
-RWBuffer<s_float4_sampler> rwb_struct_objs; /* expected-error {{'SamplerState' is an object and cannot be used as a type parameter}} expected-note {{usage of 'SamplerState' found in field 'f_sampler' of type 's_float4_sampler'}} fxc-pass {{}} */
+RWBuffer<SamplerState> rwb_struct_objs; /* expected-error {{'SamplerState' is an object and cannot be used as a type parameter}} fxc-pass {{}} */
```

Comment:
Unless you tested FXC, we should probably remove the `fxc-pass {{}}` part.
```suggestion
RWBuffer<SamplerState> rwb_struct_objs; /* expected-error {{'SamplerState' is an object and cannot be used as a type parameter}} */
```

Reply:
It was pre-existing. I'll remove it.

Reply:
Applied in new commit.